noir_stdlib/src/field/bn254.nr

```diff
@@ -74,7 +79,12 @@ pub fn decompose(x: Field) -> (Field, Field) {
 
 pub fn assert_gt(a: Field, b: Field) {
     if is_unconstrained() {
-        assert(unsafe { field_less_than(b, a) });
+        assert(
+            unsafe {
+                //@safety : unsafe in unconstrained*
```

Comment:
I'm not sure what this and the following "unsafe in unconstrained" means: does the `assert` make it safe? what's unsafe?

Also: spurious `*`
```suggestion
                //@safety : unsafe in unconstrained
```

Reply:
It's because this code is unconstrained due to the test above. I rephrased it to 'already unconstrained', it should be clearer?